doc/git-integration.md

```diff
@@ -347,3 +360,39 @@ calendar servers:
     gpg:                using RSA key 6399011044E8AFB2
     gpg: Good signature from "Peter Todd <pete@petertodd.org>"
     gpg:                 aka "[jpeg image of size 5220]"
+
+
+Configuration
+-------------
+
+The OpenTimestamps GPG wrapper can be configured in the following ways:
+
+
+```bash
+# Disable OpenTimestamps for the current repository:
+git config opentimestamps.enable false
```

Comment:
Would it be more useful to disable opentimestamps selectively for just log viewing?

Also, can you set `gpg.program` on a per-repo basis?

Reply:
> Also, can you set gpg.program on a per-repo basis?

Yes, you can. Without `--global`, `git config` operates only per-repo.

> Would it be more useful to disable opentimestamps selectively for just log viewing?

Sure we can add something like `git config opentimestamps.on_log` and `git config opentimestamps.on_commit` or `git config commit.opentimestamps` etc. for more fine-grained control. But AFAIK it is not so simple or elegant to find the current `git` operation from within the gpg wrapper. We could find the parent `git` process' command-line and parse it. Doing this in a shell wrapper is not nice, however. Another argument to making `ots-git-gpg-wrapper.sh` a Python script (or finding a way to have `ots-git-gpg-wrapper` do this already.)